src/assets/icons/icons.svg

```diff
@@ -0,0 +1,69 @@
+    </symbol>
+    <symbol id="icon-arrow-up" viewBox="0 0 12 8">
+      <desc>Arrow Up Icon</desc>
+      <path d="M5.76.81L.1 6.546a.346.346 0 000 .486.335.335 0 00.48 0L6 1.541l5.42 5.49a.335.335 0 00.48 0 .348.348 0 00.1-.242.338.338 0 00-.1-.242L6.238.813A.335.335 0 005.76.811z"/>
+    </symbol>
+    <symbol id="icon-calendar" viewBox="0 0 12 12">
+      <desc>Calendar Icon</desc>
+      <path d="M11.562 2.342a1.941 1.941 0 00-.524-.981 1.567 1.567 0 00-.275-.222l-.013-.014c-.013-.013-.026-.013-.039-.026a.532.532 0 00-.118-.065h-.013A1.935 1.935 0 009.664.81h-.811V.25a.248.248 0 10-.497 0v.536H3.658V.25A.257.257 0 003.409 0a.248.248 0 00-.249.249v.536H2.35c-.249 0-.497.053-.72.144a2.036 2.036 0 00-.654.432 2.236 2.236 0 00-.275.34 2.32 2.32 0 00-.196.406c-.026.078-.052.157-.066.235-.026.131-.039.262-.039.393v7.341A1.92 1.92 0 002.323 12h7.354a1.92 1.92 0 001.924-1.924V2.735c0-.13-.013-.262-.04-.393zM.91 2.722c0-.105.013-.196.026-.288.013-.092.052-.183.078-.275.053-.118.118-.222.183-.314.04-.052.079-.091.118-.144.079-.078.17-.144.262-.21.222-.13.471-.209.746-.209h.811v.537a.248.248 0 10.497 0v-.537h4.764v.537a.248.248 0 10.497 0v-.537h.811c.275 0 .537.079.746.21.092.065.184.13.262.21a1.386 1.386 0 01.419 1.02v1.073H.91V2.722zm10.233 7.341c0 .406-.157.772-.419 1.034a1.445 1.445 0 01-1.02.419H2.348c-.393 0-.759-.157-1.02-.419a1.445 1.445 0 01-.42-1.02V4.278h10.234v5.784z"/>
+    </symbol>
+    <symbol id="icon-close" viewBox="0 0 18 18">
+      <desc>Close Icon</desc>
+      <path d="M15.364 2.636A8.936 8.936 0 009 0a8.936 8.936 0 00-6.364 2.636A8.936 8.936 0 000 9c0 2.405.936 4.663 2.636 6.364A8.936 8.936 0 009 18a8.936 8.936 0 006.364-2.636A8.936 8.936 0 0018 9a8.936 8.936 0 00-2.636-6.364zM14.64 14.64A7.926 7.926 0 019 16.977a7.926 7.926 0 01-5.64-2.337C.25 11.53.25 6.47 3.36 3.36A7.926 7.926 0 019 1.023c2.129 0 4.133.83 5.64 2.337 3.11 3.11 3.11 8.17 0 11.28z M12.966 5.034a.51.51 0 00-.724 0L9 8.276 5.757 5.034a.51.51 0 00-.723 0c-.2.2-.2.523 0 .724L8.276 9l-3.242 3.242a.515.515 0 00.36.875.5.5 0 00.36-.151l3.242-3.243 3.242 3.243a.515.515 0 00.36.151c.133 0 .262-.049.36-.151.2-.2.2-.523 0-.724L9.723 9l3.243-3.242c.2-.201.2-.523 0-.724z"/>
+    </symbol>
+    <symbol id="icon-copy" viewBox="0 0 12 12">
+      <desc>Copy Icon</desc>
+      <path d="M7.723 2.099H2.144a.95.95 0 00-.948.948v8.004a.95.95 0 00.948.949h5.579a.95.95 0 00.949-.949V3.047a.952.952 0 00-.95-.948zm.283 8.95a.285.285 0 01-.286.285H2.142a.285.285 0 01-.285-.285V3.047c0-.157.128-.285.285-.285H7.72c.158 0 .286.128.286.285v8.002z M9.856 0H4.278a.95.95 0 00-.949.949.33.33 0 00.332.331.33.33 0 00.332-.331c0-.158.127-.285.285-.285h5.578c.158 0 .285.127.285.285v8.004a.285.285 0 01-.285.285.33.33 0 00-.332.332.33.33 0 00.332.331.95.95 0 00.949-.948V.949A.95.95 0 009.856 0z"/>
+    </symbol>
+    <!-- REFACTOR EDIT ICON -->
```

Comment:
@felipprodrigues did you refactor already?

Reply:
This icon is no longer being used